lua/entities/gmod_wire_expression2/core/bone.lua

```diff
@@ -47,9 +47,9 @@ end
 E2Lib.getBone = getBone
 
 local function GetBones(entity)
-	if not entity2bone[entity] then
-		entity2bone[entity] = {}
-		for i = 0, entity:GetPhysicsObjectCount() - 1 do
+	local bone_count = entity:GetPhysicsObjectCount()
+	if table.Count(entity2bone[entity] or {}) ~= bone_count then
```

Comment:
Are an entity's bones count able to change? I'm not sure what this fixes.

Reply:
`entity2bone[entity]` exists when you get only 1 bone, so this assumes the array is full. Not sure how to do this elegantly.